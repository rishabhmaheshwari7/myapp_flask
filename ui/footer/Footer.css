.footer-design{
    color:white;
    background: #282763;
    /* padding-top: 0; */
    bottom:0;
    width:100%;
    font-size: larger;
    height: 200px;
    position:relative;
    display: flex;
    flex-direction: column;
    flex-grow: 0;
    flex-shrink: 0;

}
a{
    text-decoration: none;
    color:white;
}
.row{
    text-align: center;
    padding-bottom: 20px;
}
.col{
    text-align: center;
    padding-top: 50px;
}