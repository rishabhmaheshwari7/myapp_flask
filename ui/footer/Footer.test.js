import { render, screen } from "@testing-library/react"

import { MemoryRouter } from "react-router-dom"
import Footer from "./Footer.component"

describe('Footer Component', () => {
  
    it('Should render the footer text correctly', () => {
        render(<MemoryRouter><Footer /></MemoryRouter>)

        expect(screen.getAllByRole("footer")).toMatchSnapshot()
    })
  })