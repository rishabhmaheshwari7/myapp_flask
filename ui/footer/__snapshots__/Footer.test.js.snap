// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Component Should render the footer text correctly 1`] = `
Array [
  <div
    class="footer-design"
    role="footer"
  >
    <div
      class="row"
    >
      <p
        class="col"
      >
        <br />
         Leap Bootcamp Â© 
        2022
         
        <br />
        Team 9Bits 
        <br />
         All Rights Reserved |
        <a
          href="https://www.lowes.com/l/about/terms-and-conditions-of-use"
        >
          Terms of Service
        </a>
         |
        <a
          href="https://www.lowes.com/l/about/privacy-and-security-statement"
        >
          Privacy
        </a>
      </p>
    </div>
  </div>,
]
`;
